web/src/main/java/no/nav/modiapersonoversikt/rest/person/PersonsokController.kt

```diff
@@ -376,9 +376,9 @@ fun PersonsokRequest.tilPdlKriterier(clock: Clock = Clock.systemDefaultZone()):
     }
 
     return listOf(
-        PdlKriterie(PdlFelt.NAVN, navn),
+        PdlKriterie(PdlFelt.NAVN, navn, searchHistorical = true),
```

Comment:
Om jeg forstår doc'en til PDL riktig så burde være 
```suggestion
        PdlKriterie(PdlFelt.NAVN, navn, searchHistorical = null),
```
PDL sin doc:
```
    # Søk i historiske data
    # true = søker kun i historiske data.
    # false = søker kun i gjeldende data.
    # null = søke i både historiske og gjeldende data.
    searchHistorical:Boolean
```